skimage/measure/fit.py

```diff
@@ -868,9 +869,13 @@ def ransac(data, model_class, min_samples, residual_threshold,
                 break
 
     # estimate final model using all inliers
-    if best_inliers is not None:
+    if best_inliers is not None and any(best_inliers):
         # select inliers for each data array
         data_inliers = [d[best_inliers] for d in data]
         best_model.estimate(*data_inliers)
+    else:
+        best_model = None
+        best_inliers = None
+        warn("User warning: no inliers found. Model not fitted")
```

Comment:
```suggestion
        warn("No inliers found. Model not fitted")
```

Reply:
`UserWarning` is the default category of `warnings.warn` so no need to mention `UserWarning` in the message (see https://docs.python.org/2/library/warnings.html#warnings.warn).